app/views/bgeigie_imports/awaiting_response/index.html.haml

```diff
@@ -22,7 +22,10 @@
           %td= link_to bgeigie_import.user.name, bgeigie_import.user
           %td= link_to File.basename(bgeigie_import.source.filename), bgeigie_import
           - contact = bgeigie_import.uploader_contact_histories.order(:created_at).first
-          %td= contact.administrator.name
+          -# = TODO: Cause crash in case where adminstrator is null
+          - if contact.administrator.nil?
+          - else
+            %td= contact.administrator.name
```

Comment:
Surprised that this would end up nil, but we probably shouldn't do it this way since we'll have unbalanced row cells.

Would use `%td= contact.administrator.try(:name)`